src/components/ExpandedNode/ExpandedNode.tsx

```diff
@@ -373,12 +396,18 @@ const ExpandedNodeMemo: FC<ExpandedNodeMemoProps> = memo(
             {guidance || branch}
           </tbody>
         </table>
+        {/* Node is advanceable if it has been accepted or declined */}
+        {pathwayState.transitions.length > 0 && !isActionable && isGuidance && isCurrentNode && (
+          <button className={`${indexStyles.button} ${styles.button}`} onClick={onAdvance}>
+            Advance
+          </button>
```

Comment:
We are switching to the material-ui button to be consistent with the builder
```suggestion
          <Button 
               className={`${indexStyles.button} ${styles.button}`} 
               onClick={onAdvance}
               variant="contained"
               color="primary"
          >
            Advance
          </Button>
```

Reply:
Thanks, I think I was copying how the default text button was styled... I'll update that button to material-ui also.

Reply:
Don't worry about updating the other buttons - I updated them in my PR